pkg/image/registry/imagestream/strategy.go

```diff
@@ -156,12 +156,6 @@ func (s Strategy) tagsChanged(old, stream *api.ImageStream) fielderrors.Validati
 		}
 
 		if tagRef.From.Kind == "DockerImage" && len(tagRef.From.Name) > 0 {
-			event, err := tagReferenceToTagEvent(stream, tagRef, "")
-			if err != nil {
-				errs = append(errs, fielderrors.NewFieldInvalid(fmt.Sprintf("spec.tags[%s].from", tag), tagRef.From, err.Error()))
-				continue
-			}
-			api.AddTagEventToImageStream(stream, tag, *event)
```

Comment:
@ncdc I'm still wondering about this change, though. Without it when importing tags I'm getting two events for external repository:
```
2.2	openshift/ruby-22-centos7	9 minutes ago	docker.io/openshift/ruby-22-centos7@sha256:5dc0...
					            9 minutes ago	openshift/ruby-22-centos7
```
First is created by above code which is not able to decompose the full spec at the time, because it wasn't imported yet, the second shows that actual import and since it was resolved in the mean time, creates a new tag event, because the event has changed. Is that ok?